tools/import-image-tests

```diff
@@ -37,7 +31,7 @@ def list_tests(source) -> Iterator[Tuple[str, str]]:
         yield name, fullpath
 
 
-def load_test(name, fullpath) -> dict:
+def load_test(name: str, fullpath: Union[str, pathlib.Path], filters: Dict[str, List[str]]) -> Dict[str, Any]:
```

Comment:
Maybe we need to define some aliases for some of the types (like we do in osbuild for the Path related things), because this is now really hard to read with all the type parameters.